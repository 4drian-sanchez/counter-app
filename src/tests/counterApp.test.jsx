import { render, screen } from "@testing-library/react"
import CounterApp from "../counterApp"
//Tarea: 
/**
 * Debe de hacer match en el snapshot
 * Debe de mostrar el valor inicial de 100
 */

describe('Pruebas en el comp <CounterApp/>', () => { 

    const value = 100;
    
    test('Debe de hacer match en el snapshot', () => { 
        const {container} = render( <CounterApp value={value}/>)
        expect( container ).toMatchSnapshot();
     })
    
    test('Debe de mostrar el valor inicial de 100', () => { 
        render( <CounterApp value={value}/>)
        expect( screen.getByText(0) ).toBeTruthy();
     })
 })