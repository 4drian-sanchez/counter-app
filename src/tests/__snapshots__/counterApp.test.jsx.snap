// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el comp <CounterApp/> Debe de hacer match en el snapshot 1`] = `
<div>
  <h1>
    CounterApp
  </h1>
  <p
    class="number"
  >
    0
  </p>
  <button>
     +1
  </button>
  <button>
    -1
  </button>
  <button>
    Reset
  </button>
</div>
`;
